src/SFA.DAS.FAT.Web/Models/StandardViewModel.cs

```diff
@@ -42,7 +48,10 @@ public static implicit operator StandardViewModel(StandardModel source)
             RouteCode = source.RouteCode,
             MaxFunding = source.MaxFunding,
             TypicalDuration = source.TypicalDuration,
-            ApprenticeshipType = source.ApprenticeshipType
+            ApprenticeshipType = source.ApprenticeshipType,
```

Comment:
We could have just changed the type of ApprenticeType to a string and used GetDescription here